include/eve/detail/function/simd/common/friends.hpp

```diff
@@ -24,6 +24,9 @@ namespace eve
   EVE_REGISTER_CALLABLE(is_not_equal_)
 }
 
+// This file needs tag::if_not_equal_ to be visible
```

Comment:
Just do a predeclaration for convert somewhere here, no?

Reply:
Yeah better